### ISSUE TITLE
macOS issue

### ISSUE THREAD

**Issue by davidpr** [dharmatech/net-liquidity.ps1#2]
I've run it on macOS by installing powershell and using pwsh command in terminal, but there's an error arising:
---
ParserError: /Users/david/Documents/marketsScripts/net-liquidity.ps1:209
Line |
 209 |  … t="true" class="Header-link js-details-target btn-link">    <svg aria …
     |                                                                ~
     | The '<' operator is reserved for future use.
---

**Comment by dharmatech** [comment]
Hey @davidpr!

Thanks for testing it out.

Open the file and take a look at it. It sounds like maybe an HTML page somehow got downloaded instead of the PowerShell script.

See the instructions here for downloading the script:

https://github.com/dharmatech/net-liquidity.ps1#downloading-the-script

Let me know how it goes.

**Comment by davidpr** [comment]
yes, it was that. I just copied the code with the raw option. It's a very good piece of work.

I was thinking that the model captures well the SP500 movement. However, buying low doesn't warranty the the index not going more down as a result of quantitative tightening. It would be great to anticipate or at least try to anticipate to a QT for example and potentially price that also. Is there any way to include those data?

**Comment by dharmatech** [comment]
Glad you got it sorted out!

**Comment by dharmatech** [comment]
As far as anticipating QT, John Comiskey had been publishing the weekly QT projection. So you could factor that in.

Until QT policy changes, you can count on $60B drop in USTS per month. The MBS drop has had been variable.

https://twitter.com/Johncomiskey77/status/1612937600485974018?t=lIDABCckuHrxjr3TYlzrfg&s=19

**Comment by davidpr** [comment]
I see the posts of how the balance sheets works here: https://johncomiskey.substack.com/ It's a bit complex but the point would be to automate a prediction for let's say next 3 months. The automation would include also some reliable source to know the rate of QE/QT but as fas as I've been checking it seems not to be something it's disclosed very precisely in the FOMC meetings.
